colors/lighthaus.vim

```diff
@@ -346,6 +346,15 @@ call s:h("sqlKeyword",    s:cyan2,      "",   "")
 call s:h("sqlSpecial",    s:orange2,    "",   "")
 " }
 
+" ― ― ― ― ― ― ― ― ― 
+" ―  VIMWIKI ― 
```

Comment:
Please add the source of the plugin below the name
https://github.com/vimwiki/vimwiki